cypress/utils/apps/query_enhancements/constants.js

```diff
@@ -21,6 +21,8 @@ export const DS_API = {
 };
 export const DSM_API = '/internal/data-source-management/fetchDataSourceMetaData';
 
+export const DELETE_API = '/api/enhancements/jobs';
```

Comment:
unrelated to this PR but not a big fan of the multiple constants defined instead of using nested constants for similar things like APIs, when issue is the number of modifications increasing as I will have to import each constant individually.

Reply:
+1 to one const, and ideally API paths are resources not actions, it should not be `DELETE_API` but something like `JOBS_API` if you want to name this specific path

Reply:
Updated in latest commit